### PR TITLE
Since instead of From?

### PR DESCRIPTION
**Pull request by vmoutoussamy**
Since instead of FRom?

# Description

Short summary of why you created this PR

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [ ] Manager-4.1
- [ ] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/installation/pages/install-intro.adoc [hunk 1]
```diff
@@ -11,7 +11,7 @@ This book provides guidance on installing {productname}.
 
 ifeval::[{suma-content} == true]
 
-From SLES{nbsp}15{nbsp}SP1, {susemgr} Server and Proxy are available as base products, and can be installed with the SLES Unified Installer.
+Since SLES{nbsp}15{nbsp}SP1, {susemgr} Server and Proxy are available as base products, and can be installed with the SLES Unified Installer.
```
Comment (Loquacity) [inline modules/installation/pages/install-intro.adoc:14]: How about this?

```suggestion
In SLES{nbsp}15{nbsp}SP1 and later, {susemgr} Server and Proxy are available as base products, and can be installed with the SLES Unified Installer.
```

**Comment by vmoutoussamy** [comment]
Yes, I prefer your suggestion : ) however this made me think... is it possible to install SUMA4.1 with 15SP1? I don't think so right? 
So maybe the "15SP1" part could cause confusion? I think this is a general information statement but can't people misunderstand it and think that 4.1 can be installed with "15SP1 and later"? 

Sorry for the trouble, but I thought it is worth to mention it.

**Comment by Loquacity** [comment]
> Yes, I prefer your suggestion : ) however this made me think... is it possible to install SUMA4.1 with 15SP1? I don't think so right?
> So maybe the "15SP1" part could cause confusion? I think this is a general information statement but can't people misunderstand it and think that 4.1 can be installed with "15SP1 and later"?
> 
> Sorry for the trouble, but I thought it is worth to mention it.

Hrm, I think they can, actually. But we can check.

**Comment by Loquacity** [comment]
> > Yes, I prefer your suggestion : ) however this made me think... is it possible to install SUMA4.1 with 15SP1? I don't think so right?
> > So maybe the "15SP1" part could cause confusion? I think this is a general information statement but can't people misunderstand it and think that 4.1 can be installed with "15SP1 and later"?
> > Sorry for the trouble, but I thought it is worth to mention it.
> 
> Hrm, I think they can, actually. But we can check. Or we can just remove the conditional bit an say "SUSE Manager Server and Proxy are installed using the unified installer" (or similar).

File: modules/installation/pages/install-intro.adoc [hunk 2]
```diff
@@ -11,7 +11,7 @@ This book provides guidance on installing {productname}.
 
 ifeval::[{suma-content} == true]
 
-From SLES{nbsp}15{nbsp}SP1, {susemgr} Server and Proxy are available as base products, and can be installed with the SLES Unified Installer.
+In SLES{nbsp}15{nbsp}SP1 and later, {susemgr} Server and Proxy are available as base products, and can be installed with the SLES Unified Installer.
```
Comment (Loquacity) [inline modules/installation/pages/install-intro.adoc:14]:
```suggestion
{susemgr} Server and Proxy are available as base products, and can be installed with the SLES Unified Installer.
```

**Comment by Loquacity** [comment]
@vmoutoussamy If you're happy with this now, let's merge it.

**Pull request merged**